examples/kubernetes/kuma-demo/README.md

```diff
@@ -0,0 +1,235 @@
+customresourcedefinition.apiextensions.k8s.io/dataplaneinsights.kuma.io created
+customresourcedefinition.apiextensions.k8s.io/dataplanes.kuma.io created
+customresourcedefinition.apiextensions.k8s.io/meshes.kuma.io created
+customresourcedefinition.apiextensions.k8s.io/proxytemplates.kuma.io created
+customresourcedefinition.apiextensions.k8s.io/trafficlogs.kuma.io created
+customresourcedefinition.apiextensions.k8s.io/trafficpermissions.kuma.io created
+clusterrole.rbac.authorization.k8s.io/kuma:control-plane created
+clusterrolebinding.rbac.authorization.k8s.io/kuma:control-plane created
+role.rbac.authorization.k8s.io/kuma:control-plane created
+rolebinding.rbac.authorization.k8s.io/kuma:control-plane created
+service/kuma-injector created
+service/kuma-control-plane created
+deployment.apps/kuma-control-plane created
+deployment.apps/kuma-injector created
+mutatingwebhookconfiguration.admissionregistration.k8s.io/kuma-admission-mutating-webhook-configuration created
+mutatingwebhookconfiguration.admissionregistration.k8s.io/kuma-injector-webhook-configuration created
+```
+
+Check the pods are up and running by checking the `kuma-system` namespace
+
+```
+$ kubectl get pods -n kuma-system
+NAME                                  READY   STATUS    RESTARTS   AGE
+kuma-control-plane-7bcc56c869-lzw9t   1/1     Running   0          70s
+kuma-injector-9c96cddc8-745r7         1/1     Running   0          70s
+```
+
+In the following steps, we will be using the pod name of the `kuma-control-plane-*************` pod. Please replace any `{KUMA_CP_POD_NAME}` with your pod name.
+
+6. Navigate into the directory where all the kuma-demo YAML files are:
+
+```
+cd kuma/examples/kubernetes/kuma-demo/
+```
+
+1. Deploy Kuma's sample marketplace application
+
+```
+$ kubectl apply -f kuma-demo-aio.yaml
+namespace/kuma-demo created
+serviceaccount/elasticsearch created
+service/elasticsearch created
+replicationcontroller/es created
+deployment.apps/redis-master created
+service/redis-master created
+service/kuma-demo-api created
+deployment.apps/kuma-demo-app created
+```
+
+This will deploy our demo marketplace application split across 3 pods. The first pod is an Elasticsearch service that stores all the items in our marketplace. The second pod is a Redis service that stores reviews for each item. The last pod is our Node/Vue application that allows you to visually query the Elastic and Redis endpoints.
+
+Check the pods are up and running by checking the `kuma-demo` namespace
+
+```
+kubectl get pods -n kuma-demo
+NAME                             READY   STATUS    RESTARTS   AGE
+es-pkm29                         2/2     Running   0          7m23s
+kuma-demo-app-5b8674794f-7r2sf   3/3     Running   0          7m23s
+redis-master-6b88967745-8ct5c    2/2     Running   0          7m23s
+```
+
+In the following steps, we will be using the pod name of the `kuma-demo-app-*************` pod. Please replace any `{KUMA_DEMO_APP_POD_NAME}` with your pod name.
```

Comment:
Don't leave setting `KUMA_DEMO_APP_POD_NAME ` up to the users.

At least, provide a copy-and-paste example how to set it, e.g.

```
KUMA_DEMO_APP_POD_NAME=$( kubectl -n kuma-demo get pods -l app=kuma-demo-api -o=jsonpath='{.items[0].metadata.name}' )
```